server/tests/unit/clientmanager.spec.ts

```diff
@@ -181,10 +181,11 @@ describe("MonolithId", () => {
 		balancerManager.addBalancerConnection(mock1);
 		balancerManager.addBalancerConnection(mock2);
 
-		const id1 = mock1.sendMock.mock.calls[0][0].payload.id;
-		const id2 = mock2.sendMock.mock.calls[0][0].payload.id;
+		const id1 = mock1.sendMock.mock.calls[0][0].filter((msg: MsgM2B) => msg.type == "init")[0]
+			.payload.id;
+		const id2 = mock2.sendMock.mock.calls[0][0].filter((msg: MsgM2B) => msg.type == "init")[0]
+			.payload.id;
```

Comment:
this is filtering the wrong one. you need to filter the calls, not the call arguments

Reply:
It's ok to leave it the way it was